### DISCUSSION TITLE
PassKey support possible?

### DISCUSSION THREAD

**Discussion by zviratko** [Yubico/libfido2#624] [Ideas]
Hello,
would it be possible to support PassKeys (I am looking for a macOS implementation in particular) via this library? I realize this is going to be highly platform specific, unless there's some interface I don't know about that's generic.
My particular use case is with SSH.

**Comment by martelletto** [comment]
In principle, yes. As you observed, it will largely depend on the system APIs involved. I have yet to take a look, so I can't elaborate further on that. In such a scenario, we would most likely be the ones providing a generic/cross-platform API. While we don't have immediate plans to add PassKey support in libfido2, we wouldn't be opposed to a PR.
